packages/design-system/src/components/input/index.js

```diff
@@ -245,7 +245,7 @@ export const InputPropTypes = {
   onFocus: PropTypes.func,
   suffix: PropTypes.node,
   unit: PropTypes.string,
-  value: PropTypes.string.isRequired,
+  value: PropTypes.string,
```

Comment:
just silencing some noise, value can actually be null so this shouldn't be required.

Reply:
This seems to _cause_ noise:

```
Warning: `value` prop on `%s` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components.
```

Reply:
Reply